src/plugins/opensearch_dashboards_react/public/context/types.ts

```diff
@@ -48,3 +49,39 @@ export interface OpenSearchDashboardsReactContext<T extends OpenSearchDashboards
   Provider: React.FC<{ services?: T }>;
   Consumer: React.Consumer<OpenSearchDashboardsReactContextValue<T>>;
 }
+
+export interface DashboardListSource {
```

Comment:
Why do you need to declare this interface (and the res of the interfaces in the file) again. they have already been defined in [src/plugins/dashboard/common/types.ts](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3090/files#diff-dc3f3df85647adf7f9b405affdac458487c272a38e496c4df9fa2bd86715b15d)

Reply:
Same. will clieanup

Reply:
Correct. Duplicate def removed.